content/agent/installation-upgrade/upgrade.md

```diff
@@ -41,43 +39,48 @@ To upgrade NGINX Agent, follow these steps:
         sudo apt-get install -y --only-upgrade nginx-agent -o Dpkg::Options::="--force-confold"
         ```
 
+## Upgrade NGINX Agent to a Specific Version 
 
+To upgrade the NGINX Agent to a specific **v2.x version**, follow these steps: 
 
-## Upgrade NGINX Agent from a version less than v2.31.0
-
-To upgrade NGINX Agent, take the following steps:
+#### Steps to Upgrade: 
 
-1. Open an SSH connection to the server where you’ve installed NGINX Agent and log in.
+1. Open an SSH connection to the server running the NGINX Agent and log in. 
 
-1. Make a backup copy of the following locations to ensure that you can successfully recover if the upgrade has issues:
+1. Back up the following files and directories to ensure recovery in case of issues during the upgrade: 
```

Comment:
I don't like the new version any better

```suggestion
1. Back up the following files and directories to ensure you can restore the environment in case of issues during the upgrade: 
```